### ISSUE TITLE
Allow configuring stdlib directories

### ISSUE THREAD

**Issue by lukaszsamson** [elixir-lsp/elixir-ls#1119]
This PR added necessary support to elixir_sense https://github.com/elixir-lsp/elixir_sense/pull/277 (elixir only). We need to expose it to elixir-ls settings and make sure OTP works as well.

**Comment by lukaszsamson** [comment]
No, it's not implemented on ElixirLS side. It shouldn't be difficult. Are you willing to submit a PR?

**Comment by cbecker** [comment]
thanks. Let me know if this works https://github.com/elixir-lsp/elixir-ls/pull/1164

I haven't been able to test it, but i'll try later today

**Comment by lukaszsamson** [comment]
Nice. Now we need a PR adding the new setting to the vscode extension in https://github.com/elixir-lsp/vscode-elixir-ls/blob/5b1c42df7c4e105bf646ddc17f95e93672c51248/package.json#L66 and it should be easy to test

**Comment by cbecker** [comment]
thanks, here it goes! https://github.com/elixir-lsp/vscode-elixir-ls/pull/452

**Comment by cbecker** [comment]
I also tested this PR with neovim and it works as expected :)

**Comment by lukaszsamson** [comment]
Nice @cbecker. Merged
